network/p2p/impl-peer.go

```diff
@@ -35,9 +35,7 @@ type messageGate interface {
 
 // Peer represents a connected peer
 type peer struct {
```

Comment:
maybe a rename for this `peer` to distinguish it from `Peer`

Reply:
You asked for it :-p I renamed `peer` to `connection` which is a better name for the internal data struct in this layer. 'Peer' is how it's exposed to outside the P2P layer.